openpype/pipeline/delivery.py

```diff
@@ -299,10 +336,28 @@ def hash_path_exist(myPath):
             os.path.join(dir_path, src_file_name)
         )
 
-        dst_padding = dst_collection.format("{padding}") % index
-        dst = "{}{}{}".format(dst_head, dst_padding, dst_tail)
-        log.debug("Copying single: {} -> {}".format(src, dst))
-        _copy_file(src, dst)
+        if renumber_frame:
```

Comment:
This way we would maybe not need `renumber_frame` argument at all? @antirotor thoughts?
```suggestion
        if re_frame_value != 0:
```

Reply:
it is the *new frame start*